### DISCUSSION TITLE
Emojis for Main Characters?

### DISCUSSION THREAD

**Discussion by syvwlch** [syvwlch/Data-Ignota#25] [Ideas]
Looking ahead to making network graphs, I'm going to need some way to indicate which node is which characters as concisely as possible.

Colors, numbers with a key off to the side would work, but will not be intuitive or scale very well.

I was thinking it'd be great to have a consistent mapping of Emojis to the most important characters instead?

**Current Candidates:**

- 🫡 : Lt. Aimer (me)
- 🌸 : Ando (me)
- 🎙️ : 7th Anonymous (me)
- ❔ : 9th Anonymous (Jay)
- 🚀 : Apollo Mojave (me)
- 🐶 : Boo (me)
- 🧸 : Bridger (Jay)
- 🙏 : Carlyle (me)
- 💣 : Casimir (Jay)
- 🔭 : Cato (Jill)
- 👸 : Danaë (me)
- 🤺 : Dominic (me)
- 💡 : Eureka (Jay)
- 🧠 : Felix (me)
- 🌞 : Ganymede (me)
- ✝️ : Heloïse (Jay)
- 🧔🏻‍♀️ : Jehova (me)
- 🕷️ : Julia (me)
- 💊 : Kosala (Jay)
- 🛟: Lesley (me)
- 💄 : Madame (me)
- 🪖 : the Major (Jill)
- 🧱 : MASON (Jay)
- 🍪 : Mommadoll (me)
- 🐜 : Mushi Mojave
- ✍️ : Mycroft (me)
- 🪒 : Ockham (me)
- 🔎 : Papa (Jill)
- 🐉: Ráðsviðr (me)
- 🫥 : Reader (me)
- 🐺 : Saladin (me)
- 🎯 : Sniper (Jay, of course)
- 👑 : Spain (Jay)
- 🦨 : Thisbe (me)
- 🧮 : Vivien (Jill)

**Missing Main Characters:**

- Martin
- Lesley
- Su-Hyeon

**Comment by syvwlch** [comment]
Created an issue [here](https://github.com/syvwlch/Data-Ignota/issues/26) to track implementation, but let's track ideas for individual emojis here?

**Comment by syvwlch** [comment]
Proposals from the Signal convo:

- 🫡 : Lt. Aimer (me)
- 🌸 : Ando (me)
- 🎙️ : 7th Anonymous (me)
- ❔ : 9th Anonymous (Jay)
- 🚀 : Apollo Mojave (me)
- 🦴 : Boo (me)
- 🧸 : Bridger (Jay)
- 🙏 : Carlyle (me)
- 💣 : Casimir (Jay)
- 🔭 : Cato (Jill)
- 👸 : Danaë (me)
- 🤺 : Dominic (me)
- 💡 : Eureka (Jay)
- 🧠 : Felix (me)
- 🌞 : Ganymede (me)
- ✝️ : Heloïse (Jay)
- 🧔🏻‍♀️ : Jehova (me)
- 🕷️ : Julia (me)
- 💊 : Kosala (Jay)
- 🛟: Lesley (me)
- 💄 : Madame (me)
- 🪖 : the Major (Jill)
- 🧱 : MASON (Jay)
- 🍪 : Mommadoll (me)
- 🐜 : Mushi Mojave
- ✍️ : Mycroft (me)
- 🪒 : Ockham (me)
- 🔎 : Papa (Jill)
- 🐉: Ráðsviðr (me)
- 🫥 : Reader (me)
- 🐺 : Saladin (me)
- 🎯 : Sniper (Jay, of course)
- 👑 : Spain (Jay)
- 🦨 : Thisbe (me)
- 🧮 : Vivien (Jill)

**Reply by cdrigby** [reply]
Looks good. As I said elsewhere, tossing this to our readers' group is a good engagement technique.
